config/helm/crds/bootstrap/kubeadmconfigs.bootstrap.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,1315 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: capi-webhook-system/capi-kubeadm-bootstrap-serving-cert
+    controller-gen.kubebuilder.io/version: v0.2.9
+  creationTimestamp: null
+  labels:
+    app.giantswarm.io/branch: '{{ .Values.project.branch }}'
+    app.giantswarm.io/commit: '{{ .Values.project.commit }}'
+    app.kubernetes.io/instance: '{{ .Release.Name }}'
+    app.kubernetes.io/managed-by: '{{ .Release.Service }}'
+    app.kubernetes.io/name: cluster-api
+    app.kubernetes.io/version: '{{ .Chart.AppVersion }}'
+    cluster.x-k8s.io/provider: bootstrap-kubeadm
+    cluster.x-k8s.io/v1alpha2: v1alpha2
+    cluster.x-k8s.io/v1alpha3: v1alpha3
```

Comment:
Do we need to add `cluster.x-k8s.io/v1beta1: v1alphaX_v1beta1`?

Reply:
This is still old v1alpha3 (CAPI v0.3.x).